asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -418,10 +431,16 @@ def main():
 
     # Parse symlinks in output_dir
     output_dir = Path(args.output_dir)
+    logger = FileLogger("run_docking_oe", path=str(output_dir)).getLogger()
+    logger.info("Starting run_docking_oe")
+    logger.info(f"Output directory: {output_dir}")
```

Comment:
does this not cause issues when the directory doesn't exist? seems like it'll try to create a file in a non-existent directory and panic

Reply:
Good point, I have added a check to see if the directory exists otherwise create it.